tests/test_command_line.py

```diff
@@ -1171,14 +1172,7 @@ def test_generate_c_source_output_directory(self):
         database_h = os.path.join(output_directory, f'{database}.h')
         database_c = os.path.join(output_directory, f'{database}.c')
 
-        if os.path.exists(database_h):
-            os.remove(database_h)
-
-        if os.path.exists(database_c):
-            os.remove(database_c)
-
-        if os.path.exists(output_directory):
-            os.rmdir(output_directory)
+        shutil.rmtree(output_directory)
```

Comment:
hm, this seems quite dangerous and unexpected behavior to me because I suppose most users won't expect the output directory to be deleted. I propose to simply bail out with an error message if the output directory already exists. `git clone` does the same.

Reply:
This is in the tests, not the actual command.  Were your concerns related to the running of the tests?

Reply:
ah I see. I just didn't notice this. sorry.